desc/basis.py

```diff
@@ -1412,3 +1416,121 @@ def zernike_norm(l, m):
 
     """
     return np.sqrt((2 * (l + 1)) / (np.pi * (1 + int(m == 0))))
+
+
+def FourierZernike_to_PoincareZernikePolynomial(X_lmn_3D, basis_3D):
```

Comment:
@dpanici remove these for now and open a new branch. 

Also: 
remove poincare stuff
make basis conversion method of basis
FourierZernike -> Zernike
FourierZernike -> Fourier
Zernike -> FourierZernike
Fourier -> FourierZernike